reown-wc/src/utils/CosmosRequestHandler.ts

```
@@ -28,6 +28,10 @@ export async function approveCosmosRequest(
       )
       return formatJsonRpcResult(id, signedAmino.signature)
 
+    case COSMOS_SIGNING_METHODS.COSMOS_GET_ACCOUNTS:
+      const accounts = await wallet.getAccounts()
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)